core/com.b2international.snowowl.core.rest/pom.xml

```diff
@@ -103,8 +99,9 @@
 									<directory>snow-owl-api-docs/build</directory>
 								</fileset>
 							</filesets>
-            			</configuration>
-          			</execution>
+							<skip>true</skip>
```

Comment:
How about this cleaning step? Should it remain skipped?

Reply:
No, this was only to prevent failures in local builds running inside Eclipse. Thanks!